variable/economy/economy.yaml

```diff
@@ -4,6 +4,18 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Electricity:
```

Comment:
How is that different from `Final Energy|Electricity`? Or shouldn't this be expenditure rather than consumption?

Reply:
I cannot seem to locate the "Final Energy|Electricity" variable in the economy.yaml file.
Am I looking at some wrong version?

Reply:
It's in the file `energy/energy-final.yaml`